vcpkg.json

```diff
@@ -53,12 +65,57 @@
     "protobuf",
     "pthreads",
     "qt5compat",
-    "qtbase",
+    {
+      "name": "qtbase",
+      "default-features": false,
+      "features": [
+        "brotli",
+        "concurrent",
+        "dbus",
+        "default-features",
+        "doubleconversion",
+        "freetype",
+        "gui",
+        "harfbuzz",
+        "icu",
+        "jpeg",
+        "network",
+        "opengl",
+        {
+          "name": "gles3",
+          "platform": "ios"
+        },
+        {
+          "name": "openssl",
+          "platform": "!ios"
+        },
+        "pcre2",
+        "png",
+        "sql",
+        {
+          "name": "sql-psql",
+          "platform": "!ios"
+        },
+        "sql-sqlite",
+        "testlib",
+        "thread",
+        "widgets",
+        "zstd"
+      ]
+    },
```

Comment:
On iOS we build with OpenGL ES 3.0 and without Postgres. This unfortunately requires us to list all of the default features explicitly(https://github.com/microsoft/vcpkg/issues/11097).